code/online_challenges/src/codechef/TREEROOT/TREEROOT.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+
+int main(void) {
+	
+	int t, n, id, cid, ans;
+	scanf("%d", &t);
+	
+	while(t--)
+	{
+	    scanf("%d", &n);
+	    ans = 0;
+	    for(int i = 0; i < n; i++)
```

Comment:
Should be preincrement. `i` should be declared above this loop. Also reformat this file using clang format and 4 spaces.